src/Cache/Service/Warm.php

```diff
@@ -6,29 +6,28 @@
 
 namespace Jstewmc\Gravity\Cache\Service;
 
-use Jstewmc\Gravity\Cache\Data\Cache;
 use Jstewmc\Gravity\{Deprecation, Id, Path, Service};
 use Psr\Log\LoggerInterface as Logger;
+use Psr\SimpleCache\CacheInterface;
```

Comment:
We could use - `use Psr\SimpleCache\CacheInterface as Cache` - for a little more brevity. I did the same with the `Psr\Log\LoggerInterface as Logger` (at least when I realized I could haha).

Reply:
I am a big fan of aliasing imports. I wasn't sure what you preferred but +1